README.md

```diff
@@ -124,33 +143,23 @@ getInfoRequest
 
 > If you are not interested in decode your JSON response into a Model you can invoke `request.rx.anyObject()` which returns an `Observable` of `AnyObject` for the current request and propagates a `OperaError` error through the result sequence if something goes wrong.
 
-> Opera can be used along with [RxAlamofire](https://github.com/RxSwiftCommunity/RxAlamofire).
-
-
-Opera represents pagination request through `PaginationRequestType` protocol which also conforms to `URLRequestConvertible`. Typically we don't need to create a new type to conform to it. Opera provides `PaginationRequest<Element: OperaDecodable>` generic type that can be used in most of the scenarios.
-
-One of the requirements to adopt `PaginationRequestType` is to implement the following initializer:
-
-```swift
-init(route: RouteType, page: String?, query: String?, filter: FilterType?, collectionKeyPath: String?)
-```
-so we create a pagination request doing:
-
+## Error Handling
+If you are using the reactive helpers (which are awesome btw!) you can handle the errors on the `onError` callback which returns an `Error` that can be casted to `OperaError` for easier usage.
```

Comment:
Sounds like any error received in the `onError` will be castable to `OperaError`, in some case the error canbe something else, like `OperaSwift.UnkownError` or whatever else the user returns in the middle as the error. Maybe you can change a little this sentence: *(...) an `Error` that, **in case of Networking or Parsing issues**, can be casted to `OperaError`*

What do you think?